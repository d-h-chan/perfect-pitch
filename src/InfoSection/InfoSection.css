.infoSectionHeader {
  border-left: 4px solid yellow; 
}
