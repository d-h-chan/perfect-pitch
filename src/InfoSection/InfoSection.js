import React, { Component } from 'react';
import './InfoSection.css';


class InfoSection extends Component {
  render() {
    return (
      <>
        <h2 className="infoSectionHeader">What is perfect pitch?</h2>
        <p>
          Absolute pitch (AP), often called perfect pitch,
          is the ability of a person to identify or re-create
          a given musical note without the benefit
          of a reference tone. Applications of perfect pitch include: 
          naming individul pitches (e.g. Fâ™¯, A, G, C) played on various instruments,
          naming the key of a given piece of tonal music,
          identifying and naming all the tones of a given chord or other tonal mass,
          or accurately singing a named pitch.
        </p>
      </>
    );
  }
}

export default InfoSection;