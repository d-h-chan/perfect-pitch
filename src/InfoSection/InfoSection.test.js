import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import InfoSection from './InfoSection'

it('renders without errors', () => {
  const wrapper = shallow(<InfoSection />)
  expect(toJson(wrapper)).toMatchSnapshot()
})