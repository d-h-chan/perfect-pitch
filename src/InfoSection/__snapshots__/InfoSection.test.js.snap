// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without errors 1`] = `
<Fragment>
  <h2
    className="infoSectionHeader"
  >
    What is perfect pitch?
  </h2>
  <p>
    Absolute pitch (AP), often called perfect pitch, is the ability of a person to identify or re-create a given musical note without the benefit of a reference tone. Applications of perfect pitch include: naming individul pitches (e.g. Fâ™¯, A, G, C) played on various instruments, naming the key of a given piece of tonal music, identifying and naming all the tones of a given chord or other tonal mass, or accurately singing a named pitch.
  </p>
</Fragment>
`;
